core/src/bundle_stage.rs

```diff
@@ -1399,6 +1514,26 @@ impl BundleStage {
 
         let mut unprocessed_bundles: VecDeque<PacketBundle> = VecDeque::with_capacity(1000);
         let mut cost_model_failed_bundles: VecDeque<PacketBundle> = VecDeque::with_capacity(1000);
+        // Initialize block limits and open up last 20% of ticks to non-bundle transactions
+        let mut reserved_space = BundleReservedSpace {
+            current_bundle_block_limit: MAX_BLOCK_UNITS,
```

Comment:
can you use bank current block limit here and below? i think might break tests that force a certain block limit size?

Reply:
can prob get a bank from poh_recorder

Reply:
i initialize with appropriate limits in the tests below, none of the tests currently reference this function

but, i agree, it would be better to use current block limit, i just couldn't convince myself that there would be a bank available when this line hits

maybe i just need to wait here until a bank is availabe?

Reply:
let's just add a TODO and figure it out later, wanna get this in and running on our machines to make sure its good for release

Reply:
i checked the logs on this, and there's ~10 minutes between when this currently gets initialized and when the first bank arrives

i don't see any reason why waiting would hurt, but i'd like to think it throught some more

Reply:
ok sg